### DISCUSSION TITLE
YouTubers that should be removed from the list

### DISCUSSION THREAD

**Discussion by JoseDeFreitas** [JoseDeFreitas/awesome-youtubers#32] [General]
**Did you find a YouTuber that you think shouldn't be on the list? Use this issue to write about it.**

I'm creating this discussion because, as the list improves, many more YouTubers will be requested. I also want to point out some things I find important, such as "What is a good/bad YouTuber?", subjectivity, and the process behind the review of the YouTubers. Please refer to the [contribution guidelines](https://github.com/JoseDeFreitas/awesome-youtubers/blob/main/guidelines.md). Also, before commenting on this discussion, I recommend you to read the text below. If you have any doubt or something you want to write about the text below, use this issue too. Thank you!

## Contents

- [What's an awesome YouTuber?](#whats-an-awesome-youtuber)
- [What's a bad YouTuber?](#whats-a-bad-youtuber)
- [Subjectivity](#subjectivity)
- [How the entries are reviewed](#how-the-entries-are-reviewed)

### What's an awesome YouTuber?

A YouTuber that has good explanations about the topics they cover, a voice that you can clearly hear (better if he puts subtitles!), a nice course structure (if for beginners, not jumping right away to a complex tutorial. For advancers not reviewing basic topics but instead focusing on complex tutorials and providing challenges), more resources the people can search for in the videos, a mix of theoretical and practical, etc.
Quality is better than quantity, but (for example) a YouTuber with just 4 videos 10 minutes length about CSS is too few. There're playlists pretty educative that have just like 6 videos, but they're all more than 1 hour, for example. Quality is better than quantity, as said before (and as said in the "[awesome manifesto](https://github.com/sindresorhus/awesome/blob/main/awesome.md)"), but the more the videos/length of the videos, the more time there is to cover a topic.

### What's a bad YouTuber?

A YouTuber that skips tutorials or that doesn't explain something, tells you to just copy & paste it, has problems with the community, or is controversial in a bad setting, has too few videos and not very explanative, a YouTuber that copies or uses content from other YouTube channels (could not be badly intended, just has playlists that contain videos from other YouTubers but no videos from them), etc.

### Subjectivity

Like many other things, subjectivity is something that is almost always present. A user can consider a YouTuber is awesome, while other user can consider it is not. To "fix" this, we need to follow some "structures" or "stereotypes" about what an awesome YouTuber is. Of course, this doesn't mean everything is subjective; we can tell by watching a YouTuber's videos if they is bad or good.

### How the entries are reviewed

When someone creates an Issue or Pull request, I look into that YouTube channel. First of all, I check if the icon, category, and playlists are in concordance with what the user that requested it put in the comments or in the template. If everything is okay, I then review the channel itself by looking into the "Playlists" section and watching some videos. "some videos" depends, but I watch many of them (first-look, not watching the entire video. Not always though, some times I watch the videos entirely.) Because of this, I may get some time to accept/deny your Pull request or Issue. If there's a small problem (common a layout problem, which is something easy to do unintentionally), I fix the problems by committing to the branch the user has created in the fork. If there's a problem I think the user could fix by reading carefully through the guidelines, I tell them to fix it by themselves. Last, if it's a problem with the YouTuber itself (I think the YouTuber shouldn't be on the list), I mention the user and keep the Pull request open to discussion so the user can tell me more about the YouTuber and why did they want to add it.

## Request the removal of a YouTuber

If you consider a YouTuber shouldn't be on the list, please comment the name of the YouTuber with your opinion about why the YouTuber should be removed. This discussion will be open for people to answer you and share opinions.

**Comment by catcecilia** [comment]
this sounds like a very good idea!

**Comment by badushaebrahim** [comment]
Good way to select good content
